### DISCUSSION TITLE
NPC Aggression Timer Feature change request

### DISCUSSION THREAD

**Discussion by TheHappyPittie** [runelite/runelite#16367] [Ideas]
![image](https://user-images.githubusercontent.com/114444106/224443456-be9df0fe-a942-44fc-a438-2675560ffabd.png)
Could the overlay in the image be changed so that it only displays if the aggression lines would also be displayed. Meaning if you dont have always active enabled this error overlay would only show if something on your NPC names list was nearby.

**Comment by Nightfirecat** [comment]
We used to have a toggle for this, then users got confused when it didn't work and didn't tell them why. We have since removed that toggle because too many users don't understand why the plugin "doesn't work" when they're not told why.

**Reply by TheHappyPittie** [reply]
Annoying but fair.

**Comment by rsValliance** [comment]
Would also love a toggle to atleast make this warning "smaller", and leave it at this by default. So we could have the NPC aggression infobox (demon head) but with a red !! over it or something, and when hovered this displays. As is it takes up so much space that if I log in while doing an AFK activity I end up just having to disable the entire plugin.
